### ISSUE TITLE
Support ARC format in visualisation tool

### ISSUE THREAD

**Issue by hannakoppelaar** [WebCuratorTool/webcurator#136]
At the KB we still have a large collection of ARC files that we would like to be able to inspect using the visualisation tool. Currently, the tool only indexes files with .warc and .warc.gz suffixes. 

It seems it should be possible to support the ARC format without too much effort. It would entail widening the file filter criteria and making IndexProcessorWarc.java more general (there's already an IndexProcessorArc.java, but that's not being used and it's probably not necessary to differentiate between WARC and ARC at that level).

**Comment by leefrank9527** [comment]
1. For the indexing of the original ARCs, it's feasible to get the URLS, parent URLS, status code and document size of the crawled record. The seed urls can be determined from the historic seed urls from webapp.
2. For the modification: the recrawled patch crawled maybe WARCs. Should merge the the WARC to the ARC cellections. The indexing of these 2 kinds of format should be taken into consideration as well.
To sum up, it's feasible to index the ARCs.

**Comment by obrienben** [comment]
@hannakoppelaar do you want to consider the patching functionality when extending the tool to support ARCs? I had assumed the  harvests with ARCs were **not** recent, so perhaps out of scope for any patching?
